src/cli/schemas/oradewrc-schema.json

```diff
@@ -161,11 +162,16 @@
           "default": "./src/{schema-name}/SYNONYMS/{object-name}.sql",
           "description": "Pattern for synonyms",
           "type": "string"
+        },
+        "apex": {
+          "default": "./src/{schema-name}/APEX/{object-name}.sql",
+          "description": "Pattern for apex applications",
+          "type": "string"
         }
       }
     },
     "import.ease": {
-      "default": false,
+      "default": true,
```

Comment:
Similar to "compile.force" it used to be `true`, but then I thought that for an average user it doesn't hurt if he/she gets all objects imported rather than guessing why some object didn't import...